src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Request as a Service",
+    "version": "4.0",
+    "title": "Concur Request API Documentation"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/travelrequest/v4",
+  "tags": [
+    {
+      "name": "Request Resource",
+      "description": "Manage documents used for pre-spend authorizations within Concur Request"
+    },
+    {
+      "name": "Workflow Resource",
+      "description": "Manage workflow transitions for a Request document"
+    },
+    {
+      "name": "Travel Agency Resource",
+      "description": "Manage the configuration for travel agencies integrated with Concur Request"
+    },
+    {
+      "name": "Policy Resource",
+      "description": "Manage policy settings"
+    },
+    {
+      "name": "Expense Resource",
+      "description": "Manage expected expense entries attached to a Request"
```

Comment:
Manage Expected Expenses attached to a Request

Reply:
1/ Rename Expense Resource by Expected Expense Resource
2/ Manage expected expenses attached to a Request (no upper case on expected expense in a sentence)